src/main/java/lotto/WebMain.java

```diff
@@ -0,0 +1,74 @@
+package lotto;
+
+import com.sun.xml.internal.ws.api.model.wsdl.WSDLOutput;
+import lotto.dto.MatchResult;
+import lotto.dto.PurchaseInfo;
+import lotto.handlebar.CustomHandlebarsTemplateEngine;
+import lotto.number.LottoNumber;
+import lotto.ticket.LottoTicket;
+import lotto.ticket.LottoTickets;
+import lotto.ticket.TicketPayment;
+import lotto.ticket.WinningLottoTicket;
+import spark.ModelAndView;
+import spark.template.handlebars.HandlebarsTemplateEngine;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static spark.Spark.*;
+
+public class WebMain {
+    public static void main(String[] args) {
+        port(8080);
+
+        get("/", (req, res) -> {
+            Map<String, Object> model = new HashMap<>();
+
+            return render(model, "/index.html");
```

Comment:
```suggestion
            return render(Collections.emptyMap(), "/index.html");
```

빈Map을 이렇게 보내보면 어떨까요?